src/server.js

```diff
@@ -567,6 +568,7 @@ function initializeClient(socket, client, token, lastMessage) {
 			active: client.lastActiveChannel,
 			networks: client.networks.map((network) => network.getFilteredClone(client.lastActiveChannel, lastMessage)),
 			token: tokenToSend,
+			commands: constants.getCommands()
```

Comment:
Let's not overload initial events with too much information. This could easily be emitted as a separate event.

Reply:
Is that a problem to initialize command autocomplete later on?

Reply:
Well, it means that there would be a time period between the user being able to type commands, and the list of comamnds changing. Meaning they would initially not see all possible commands in the autocomplete. Could be confusing.

Reply:
You removed the command list from the client, so it simply won't complete any commands on initial load.

However, if you send commands in a separate event right after `init`, I don't see it being a problem (lazy load is fine by me, you don't usually type commands instantly after init).

I can also see this potentially being re-used if new commands are added while the server is running (plugin auto load, or whatever we end up doing).

Reply:
All good points.

> I can also see this potentially being re-used

Yeah, this alone is probably big enough of an advantage to warrant the small time period of having no autocomplete.

Reply:
I have moved this into it's own event. It's pretty small, but does the job. Nice and simple code :-P